docs/source/api/exporters.rst

```diff
@@ -0,0 +1,69 @@
+Exporters
+=========
+
+.. module:: nbconvert.exporters
+
+.. seealso::
+
+   :doc:`/config_options`
+     Configurable options for the nbconvert application
+
+.. autoclass:: Exporter
+
+    .. automethod:: __init__
+
+    .. automethod:: from_notebook_node
+
+    .. automethod:: from_filename
+
+    .. automethod:: from_file
+
+    .. automethod:: register_preprocessor
+
+Specialized exporter classes
+----------------------------
+
+.. autoclass:: HTMLExporter
+
+.. autoclass:: SlidesExporter
+
+.. autoclass:: TemplateExporter
```

Comment:
I think this should move up to just under Exporter - TemplateExporter is the main API point that people should inherit from to define custom exporters; it's not specialised in the same sense as the others in this section.